3rdparty/tbb/CMakeLists.txt

```diff
@@ -35,7 +39,9 @@ elseif(EXISTS "${tbb_src_dir}/tbb-${OPENCV_TBB_RELEASE_}")
 else()
   message(FATAL_ERROR "TBB: Can't configure TBB. Specify OPENCV_TBB_SUBDIR through command-line.")
 endif()
-set(tbb_src_dir "${tbb_src_dir}/${OPENCV_TBB_SUBDIR}")
+set(OPENCV_TBB_GITHUB_SUBDIR "${OPENCV_TBB_SUBDIR}")
+set(OPENCV_TBB_GITCODE_SUBDIR "tbb-${OPENCV_TBB_RELEASE}-60b7d0a78f8910976678ba63a19fdaee22c0ef65") # suffix is the commit id
```

Comment:
Can we use commit id as what is done for Tengine to avoid changing subdir for TBB? Same thing happens to ADE.

Reply:
Packages downloaded from the release page of Gitlab-style mirrors adds a 'v' to the extracted directory name. For example "ade-0.1.1f" is the extracted directory name of the package from Github, "ade-v0.1.1f-${commid_id}" is the one of the package from Gitcode. Use commit id to download ADE can avoid such tricky problems.